controllers/admin/AdminImagesController.php

```diff
@@ -774,6 +774,7 @@ public function initContent()
 
         if ($this->display == 'edit') {
             $this->warnings[] = $this->trans('After modification, do not forget to regenerate thumbnails', [], 'Admin.Design.Notification');
+            $this->warnings[] = $this->trans('Make sure the theme you use does not rely on this image format before deleting it.', [], 'Admin.Design.Notification');
```

Comment:
```suggestion
            $this->warnings[] = $this->trans('Make sure the theme you use doesn\'t rely on this image format before deleting it.', [], 'Admin.Design.Notification');
```

Reply:
let's keep it as Julie suggested :)

Reply:
Hi, we use contractions on the project to give a human touch to wordings. :)